drivers/gles2/rasterizer_storage_gles2.h

```diff
@@ -96,6 +96,9 @@ class RasterizerStorageGLES2 : public RasterizerStorage {
 		GLuint depth_type;
 		GLuint depth_buffer_internalformat;
 
+		// in some cases the legacy render didn't orphan. We will mark these
+		// so the user can switch orphaning off for them.
+		bool desire_orphan;
```

Comment:
Poor choice of verb + noun combo? :thinking:

Reply:
`should_orphan` might work fine and avoid getting us flagged by overzealous algorithms ;)